spec/requests/api/projects_spec.rb

```diff
@@ -353,6 +353,46 @@
       expect(response.content_type).to start_with("application/json")
       expect(response.body).to be_json_eql([].to_json)
     end
+
+    context "for a Go project that is not in the DB" do
+      let!(:project) { create(:project, platform: "Go", name: "known/project") }
+
+      context "that redirects to a known project" do
+        it "redirects" do
+          allow(PackageManager::Go)
+            .to receive(:project_find_names)
+            .with("unknown/project")
+            .and_return([project.name])
+
+          get "/api/go/unknown%2Fproject/contributors"
+          expect(response).to redirect_to("/api/go/known%2Fproject/contributors")
+        end
+      end
+
+      context "that redirects to an unknown project" do
+        it "redirects" do
```

Comment:
Should this be `it "returns not found"`?

Reply:
Sure seems like it